modules/ROOT/pages/faq.adoc

```diff
@@ -256,3 +256,16 @@ This YAML configuration is never directly processed by FCOS instances (only the
 Although similar, Ignition configurations and FCCT ones do not have the same structure; thus, converting between them is not just a direct YAML-to-JSON translation, but it involves additional logic.
 FCCT exposes several customization helpers (e.g. distribution specific entries and common abstractions) that are not present in Ignition and make the formats not interchangeable.
 Additionally, the different formats (YAML for FCCT, JSON for Ignition) help to avoid mixing up inputs by mistake.
+
+## What is the format of the version number?
+
+This is covered in detail in the https://github.com/coreos/fedora-coreos-tracker/blob/master/Design.md#version-numbers[design docs].
+
+The summary is that Fedora CoreOS uses the format `X.Y.Z.A`
+
+- `X` is the Fedora major version (i.e. `32`)
+- `Y` is the datestamp that the package set was snapshotted from Fedora (i.e. `20200715`)
+- `Z` is a code number used by official builds
```

Comment:
I wonder if we should put sub bullets:

```
    - `1` for our `next` stream
    - `2` for our `testing` stream
    - `3` for our `stable` stream
```

Reply:
Yeah, since those are the numbers most end-users will be seeing